tests/test_push_notifications.py

```diff
@@ -166,15 +167,68 @@ def test_publish_should_fail_if_no_interests_passed(self):
         )
         with self.assertRaises(ValueError):
             pn_client.publish(
-                    interests=[],
+                interests=[],
+                publish_body={
+                    'apns': {
+                        'aps': {
+                            'alert': 'Hello World!',
+                        },
+                    },
+                },
+            )
+
+    def test_publish_should_succeed_if_100_interests_passed(self):
+        pn_client = PushNotifications(
+            'INSTANCE_ID',
+            'SECRET_KEY'
+        )
+        with requests_mock.Mocker() as http_mock:
+            http_mock.register_uri(
+                requests_mock.ANY,
+                requests_mock.ANY,
+                status_code=200,
+                json={
+                    'publishId': '1234',
+                },
+            )
+            pn_client.publish(
+                interests=[str(el) for el in random.sample(range(1000), 100)],
```

Comment:
Having random interests seems potentially dangerous - this makes the tests non deterministic.
How about:
```
['interest-' + str(i) for i in range(0, 100)]
```